integration-tests/http/__tests__/order/admin/rma-flows.spec.ts

```diff
@@ -621,8 +597,7 @@ medusaIntegrationTestRunner({
         expect(pendingPaymentCollection).toEqual(
           expect.objectContaining({
             status: "not_paid",
-            // TODO: The payment should also include taxes
-            amount: 115.9,
+            amount: 15.9,
```

Comment:
q: shouldn't this be 227.9?

Reply:
Why 227.9? One item is being returned and another one being added. They have the same price, so it stays the same.
But good question, not sure if I need another property on the summary to display these expected values, that are not yet complete. Like returns.

Reply:
It came from the formula i knew from previously. I'll resolve this and focus on the other comment, which points to the same issue.